src/components/header/header.tsx

```diff
@@ -15,39 +21,52 @@ const Header = () => {
       id="header"
       top={0}
       py={{ sm: "xs", md: "s" }}
+      px="s"
       zIndex={999}
       bgColor="ui.white"
     >
-      <HStack
-        justify="center"
-        align="center"
-        sx={{
-          display: "flex",
-          alignSelf: "stretch",
-        }}
-      >
+      <Box alignItems="center" maxWidth="1250px" mx="auto">
         <HStack
-          justify="space-between"
-          align="center"
-          paddingLeft={{ sm: "16px", md: "9px" }}
-          paddingRight="16px"
           sx={{
-            display: "flex",
-            width: "1280px",
+            justifyContent: "space-between",
+            alignContent: "center",
           }}
+          pl="9px"
         >
-          <DCLogo />
-          <VStack py="s">
-            <Box display={{ sm: "none", md: isScrolled ? "flex" : "none" }}>
-              I am nav links
+          <DCLogo isMobile={true} />
+          <Box
+            sx={{
```

Comment:
You could move this up to the `HStack` parent so that everything is hidden on mobile, including the logo. It's more nitpicky and doesn't change the behavior, but it's the pattern for hiding what is not needed on mobile. `DCLogo` does it's own logic so I know it's not needed.